src/database/schema/ChatSchema.ts

```diff
@@ -270,7 +270,9 @@ class ChatSchema implements BaseDbSchema {
       console.log(e);
     }
     try {
-      attachmentJson = JSON.stringify(json?.message?.attachments);
+      if (json?.message?.message_type !== 'reply_prompt') {
+        attachmentJson = JSON.stringify(json?.message?.attachments);
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are logical and seem to be in line with the PR title. However, it's important to note that if `json?.message?.message_type` is 'reply_prompt', `attachmentJson` will remain undefined. If this variable is used later without checking for its existence, it could lead to potential errors. Consider setting a default value for `attachmentJson` before the conditional statement.
```diff
+    let attachmentJson = null;
     if (json?.message?.message_type !== 'reply_prompt') {
       attachmentJson = JSON.stringify(json?.message?.attachments);
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->